src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceOutputFormat.java

```diff
@@ -79,7 +79,9 @@ public void commitJob(JobContext jobContext) {
           String jobId = conf.get(SalesforceSinkConstants.CONFIG_JOB_ID);
           SalesforceBulkUtil.closeJob(bulkConnection, jobId);
         } catch (AsyncApiException e) {
-          throw new RuntimeException("There was issue communicating with Salesforce", e);
+          throw new RuntimeException(
```

Comment:
Is the cause captured here correctly? Do we need to check for IApiFault ?

Reply:
IAPIFault will work for ConnectionException